pyproject.toml

```diff
@@ -17,13 +17,15 @@ django-json-404-middleware = { git = "https://github.com/Axeltherabbit/django-js
 python-dotenv = "^0.21.0"
 drf-problems = { git = "https://github.com/Axeltherabbit/drf-problems" }
 typing-extensions = "^4.4.0"
+pytz = "^2022.6"
+tzdata = "^2022.7"
+coverage = "^6.5.0"
+drf-yasg = "^1.21.4"
 
 [tool.poetry.group.dev.dependencies]
 black = "^22.10.0"
 pyright = "^1.1.278"
-coverage = "^6.5.0"
 pycodestyle = "^2.9.1"
-drf-yasg = "^1.21.4"
```

Comment:
why were these moved to main dependencies?

Reply:
because it was erroring saying the coverage module and the drf-yasg modules were not installed. They weren't being installed because they were in the dev dependencies and you were running poetry install --only-main. As such I moved them into main so it didn't error

Reply:
off couse, you have to set your .env to use the .base settings.py.

Reply:
I don't understand. this seems a lot more complicated than just including them in the main dependencies

Reply:
uh? no, it's what DJANGO_SETTINGS_MODULE= in the .env file was made for